omero/developers/Python.rst

```diff
@@ -6,24 +6,57 @@ libraries.
 
 From OMERO 5.6.0 release, the client library ``omero-py`` supports Python 3 and
 is now available on PyPI_ and Conda_. The ``omero-py`` API documentation is available at https://omero-py.readthedocs.io/.
-We recommend you use a Python virtual environment to install the client library. You can create one using either ``venv`` or ``conda`` (preferred).
-If you opt for Conda_, you will need
-to install it first, see `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ for more details.
-
-To install ``omero-py`` using venv:
+We recommend you use a Python virtual environment to install the client library. You can create one using either ``venv``, ``conda`` or ``mamba`` .
+Before installing ``omero-py``, we recommend to install the `Zeroc IcePy 3.6 <https://zeroc.com/ice/downloads/3.6>`_ Python bindings.
+
+Our commercial partner `Glencoe Software <https://www.glencoesoftware.com/>`_ has produced several Python wheels to install the Ice-Python bindings depending on the desired Python version and the operating system.
```

Comment:
Could add something like: "Please visit the linked pages to find the full URLs to the wheels that are used in the examples below".

Reply:
Maybe also link to https://www.glencoesoftware.com/blog/2023/12/08/ice-binaries-for-omero.html rather than Glencoe's website ?